tests/command_list

```diff
@@ -356,9 +356,6 @@ fincore
 find
 findfs
 findmnt
-fish
-fish_indent
-fish_key_reader
```

Comment:
Why are these command removed ?

Reply:
These commands are removed because the CI/CD was failing from before: https://github.com/Azure/CloudShell/actions/runs/4079840732